.github/stale.yml

```diff
@@ -0,0 +1,19 @@
+# Configuration for stale probot
+# https://probot.github.io/apps/stale/
+#
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 90
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - lifecycle/frozen
```

Comment:
`lifecycle/frozen` label prevents an issue from staling. Please add it if you want an issue open forever.

it can also be added by
/label lifecycle/frozen